firedrake/mesh.py

```diff
@@ -77,7 +77,8 @@ def set(self):
             else:
                 base = self.mesh._base_mesh.exterior_facets.set
             return op2.ExtrudedSet(base, layers=self.mesh.layers)
-        return op2.Set(size, "%s_%s_facets" % (self.mesh.name, self.kind),
+        return op2.Set(size, "%s%sFacets" % (self.mesh.name,
+                                             self.kind.capitalize()[:3]),
```

Comment:
Or rather, they're either "plexmesh" or the name of a mesh file.  I don't think that's useful information.